ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -63,11 +63,11 @@ const translations = defineMessages({
     defaultMessage: ' No results found.'
   },
   previousPage: {
-    id: 'craftercms.ice.assets.previousPage',
+    id: 'pagination.previousPage',
     defaultMessage: 'previous page'
   },
   nextPage: {
-    id: 'craftercms.ice.assets.nextPage',
+    id: 'pagination.nextPage',
     defaultMessage: 'next page'
   },
```

Comment:
Make sure to match both ids and messages. Otherwise the i18n script will throw. They should match in capitalization too.